src/taxonomy/taxonomy-card/TaxonomyCard.jsx

```diff
@@ -0,0 +1,141 @@
+import React, { useState } from 'react';
+import {
+  Badge,
+  Card,
+  OverlayTrigger,
+  Popover,
+} from '@edx/paragon';
+import { injectIntl, intlShape } from '@edx/frontend-platform/i18n';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+import messages from '../messages';
+import TaxonomyCardMenu from './TaxonomyCardMenu';
+import ExportModal from '../modals/ExportModal';
+
+const TaxonomyCard = ({ className, original, intl }) => {
+  const {
+    id, name, description, systemDefined, orgsCount,
+  } = original;
+
+  const [isExportModalOpen, setIsExportModalOpen] = useState(false);
+
+  const orgsCountEnabled = () => orgsCount !== undefined && orgsCount !== 0;
+
+  const onClickMenuItem = (menuName) => {
+    switch (menuName) {
```

Comment:
If there is only one option, wouldn't this be clearer as a simple if condition?

Reply:
In future tasks more menu items will be added. I have added a comment for it

Reply:
Thanks for the suggestion. I like it. I added it on this commit: https://github.com/openedx/frontend-app-course-authoring/pull/645/commits/080b03ec36a5d019f475bf31a9a5fe5791973e9b